README.md

```diff
@@ -318,6 +318,36 @@ To detect the currently used localization in your tests, use the following code:
 You can access the language using the `deviceLanguage` variable.
 ```
 
+
+## Prefilling
+
+- You can prefill data within your app using [HSTestingBackchannel](https://github.com/ConfusedVorlon/HSTestingBackchannel)
+
+## Passing a Preprocessor Macro
+
+- Run snapshot using [Fastlane with .env files](https://github.com/fastlane/fastlane/blob/master/docs/FAQs.md#multiple-targets-of-the-same-underlying-app)
```

Comment:
I think sending a default value is a much better solution. I'll let you update runner.rb, and I'll update this request.